cfme/tests/openstack/infrastructure/test_host_lifecycle.py

```diff
@@ -16,12 +16,14 @@
 @pytest.fixture(scope='module')
 def host(provider):
     """Find a host for test scenario"""
-    view = navigate_to(OpenstackNode, 'All')
-    hosts = view.entities.get_all()
+    host_collection = provider.appliance.collections.openstack_nodes
+    hosts = host_collection.all(provider)
```

Comment:
This is where `.filter()` _would_ be used if `all()` was implemented correctly (just FYI, nothing for you to fix here unless you're feeling adventurous.

Reply:
**REQUIRED:** Please add a TODO about this here